### PR TITLE
Adds missing codeblock in Installing Oppia for Linux Documentation 

### LINKED ISSUE #230: [ Oppia Web ] Codeblock missing in installing Oppia in Linux documentation
While setting up Oppia locally on my Linux machine I found a command in the documentation which should be in a codeblock to make copying it to our terminal easier and to maintain consistency (and to prevent issues where we clip out the command while copying and it creates issues later on).

Page: https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Linux%3B-Python-3%29#setup-a-virtual-environment

image

### PR DESCRIPTION
**Pull request by LalitChauhan56**
Fixes #230 
This PR adds the missing codeblock as discussed in #230.

### CODE REVIEW

**Comment by seanlip** [comment]
Hi @LalitChauhan56, thanks for your PR. Please follow the instructions here first though, we need a CLA to accept contributions: https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up. Thanks.

**Comment by LalitChauhan56** [comment]
Hi @seanlip sorry for missing such an important step. I just signed the CLA. Should this PR be reopened or should I open a new one?

**Comment by seanlip** [comment]
We can reopen it, thanks.

File: Installing-Oppia-(Linux;-Python-3).md [hunk 1]
```diff
@@ -96,7 +96,7 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
 1. **Make sure you install the Python build dependencies for your operating system. These are specified [here](https://github.com/pyenv/pyenv/wiki#suggested-build-environment). If you don't do this it might lead to problems further on.** The build dependencies for Ubuntu/Debian are
 
 
-    `sudo apt-get install make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget llvm libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev`
+    ```sudo apt-get install make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget llvm libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev```
```
Comment (seanlip) [inline Installing-Oppia-(Linux;-Python-3).md]: Could you also ensure that this is properly displayed indented, under point "1", and similarly for the stuff in point 2? Currently the code blocks and the second paragraph of point 2 jut out into the left margin.

**Review by seanlip — CHANGES_REQUESTED** [review]

**Comment by LalitChauhan56** [comment]
image

I think this fixes the indentation issue.

**Review by seanlip — APPROVED** [review]
LGTM. Thanks!

**Comment by seanlip** [comment]
(And congrats on your first PR to Oppia :) )

**Comment by LalitChauhan56** [comment]
> (And congrats on your first PR to Oppia :) )

Thank you for helping me out! :)

**Pull request merged**